src/Constants.ts

```diff
@@ -150,6 +150,17 @@ export class Constants {
     },
   };
 
+  public static panels = {
+    log: {
+      viewType: 'truffle-vscode.panel.log',
```

Comment:
Should we convert this into `export const VIEW_TYPE = ...` or something similar to make it simpler?